### PR TITLE
conda-smithy v3.0.0

### PR DESCRIPTION
**Pull request by scopatz**
Merge only after success.

This pull request was auto-generated by [rever](https://regro.github.io/rever-docs/)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   url: https://github.com/conda-forge/conda-smithy/releases/download/v{{ version }}/conda-smithy-{{ version }}.tar.gz
-  sha256: b6e8e93de0240278d9cf2edd10bb972c5063e6f4653ecc9335f7db6b4ecfc65d
+  sha256: a2c0d27d32e5da426631f9d652f93e122692947ff3594339a01ffcd2ccf89e10
 
 build:
-  number: 1
+  number: 0
   noarch: python
   script:
     - python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should we switch to `pip`?

Comment (scopatz) [reply recipe/meta.yaml]: I can.

**Comment by scopatz** [comment]
@conda-forge/core Any ideas on how the pinnings here should be updated?  This is failing because of the current pinnings. Should they just be removed?

**Comment by jakirkham** [comment]
Also the requirements have changed. Most notably `conda` can be 4.2+. `conda-build` must be 3+. Finally `conda-build-all` is not a requirement any more. In case I missed something, ref below.

ref: https://github.com/conda-forge/conda-smithy/blob/v3.0.0/conda_smithy.recipe/meta.yaml#L22-L30

**Comment by jakirkham** [comment]
Think we posted at the same time. :) So hopefully that answers it.

Thanks for doing this @scopatz. :)

**Comment by scopatz** [comment]
No problem!  I'll fix this right after I get some coffee :)

**Comment by isuruf** [comment]
Needs a conda-smithy rerender. :smile:

**Comment by jakirkham** [comment]
Indeed. Turtles all the way down again. :)

Though the release did get out there.

**Comment by jakirkham** [comment]
Re-rendering in PR ( https://github.com/conda-forge/conda-smithy-feedstock/pull/92 ).

**Pull request merged**